style.css

```diff
@@ -106,12 +107,13 @@ img {
 
 /* Text */
 
-h1 {
+.main-headings {
 	font-family: var(--main-heading-font);
 	font-size: var(--font-xxl);
 }
 
-h2 {
+.sub-headings {
+	font-family: var(--sub-heading-font);
 	font-size: var(--font-lg);
 }
```

Comment:
Can we edit the bottom margin? Right now it's the browser default and it looks like too much space.

Reply:
added margin-bottom 0